File: Protium.Digital.Domain/Entities/Driver.cs
namespace Protium.Digital.Domain.Entities;

public class Driver
{
    public string Id { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string VehiclePlate { get; set; }
    public DateTime? StartDate { get; set; }
    public DateTime? ExpirationDate { get; set; }
    public DateTime? CreatedBy { get; set; }
    public DateTime? UpdatedBy { get; set; }
    public bool Active { get; set; }
}

File: Protium.Digital.API/client-app/src/models/driver.ts
export interface Driver {
    id: string;
    first_name: string;
    last_name: string;
    vehicle_plate: string;
    start_date: string | Date;
    expiration_date: string | Date;
    created_by: string;
    updated_by: string;
    active: boolean;
}
